cur/programming/lists/hof/ttt/06-block-opponent.html

```diff
@@ -3,26 +3,39 @@
 <head>
 <meta charset="utf-8" />
 <script src="/bjc-r/llab/loader.js"></script>
-<title>Blocking the Opponent</title>
+<title>Blocking The Opponent</title>
```

Comment:
Minor, but generally 'the' isn't capitalized in titled when it's the middle word, right?